pkg/cloud/services/network/routetables.go

```diff
@@ -415,6 +422,7 @@ func (s *Service) getRoutesToPrivateSubnet(sn *infrav1.SubnetSpec) (routes []*ec
 
 	routes = append(routes, s.getNatGatewayPrivateRoute(natGatewayID))
 	if sn.IsIPv6 {
+		routes = append(routes, s.getNat64PrivateRoute(natGatewayID))
```

Comment:
1. Not sure why this was not needed for managed installs?
2. Do we need to check for `!sn.IsPublic` here?